<launch>
	<arg name = "frequency" default = "10" />
	<node 
		type="talker"		
		name="talker"  
		pkg="beginner_tutorials"
		output="screen"
		args = "$(arg frequency)"
	/>
	
	<node
		type="listener"		
		name="listener"
		
   		pkg="beginner_tutorials"
		output="screen"
		launch-prefix = "gnome-terminal -e"
  />
</launch>
